lib/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @fileoverview Shareable eslint plugin for use by eslint-config-silvermine
+ * @author Justin Hummel
```

Comment:
We don't do `@author` file attribution in our codebases. Please remove. `git blame` provides all the info that's needed, and is much more accurate.